index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>project 2- product card Organic Fruit Juice</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <img
+        src="https://img.freepik.com/free-psd/colorful-delicious-smoothies-mock-up_23-2148237106.jpg?size=626&ext=jpg"
+        alt="organic juices mockup image"
+        width="350"
```

Comment:
Instead of using this hardcoded `width` attribute, you should control your image size and layout via CSS.